crates/superposition_types/src/database/models/cac.rs

```diff
@@ -384,3 +384,28 @@ impl DependencyGraph {
         }
     }
 }
+
+#[derive(Serialize, Clone, Debug)]
+#[cfg_attr(
+    feature = "diesel_derives",
+    derive(Queryable, Selectable, Insertable, AsChangeset)
+)]
+#[cfg_attr(feature = "diesel_derives", diesel(check_for_backend(diesel::pg::Pg)))]
+#[cfg_attr(feature = "diesel_derives", diesel(table_name = webhooks))]
+#[cfg_attr(feature = "diesel_derives", diesel(primary_key(name)))]
+pub struct Webhooks {
+    pub name: String,
+    pub description: String,
+    pub enabled: bool,
+    pub url: String,
+    pub method: String,
+    pub version: String,
+    pub custom_headers: Option<Value>,
+    pub events: Vec<String>,
+    pub max_retries: i32,
+    pub last_triggered_at: Option<NaiveDateTime>,
+    pub created_by: String,
+    pub created_at: NaiveDateTime,
+    pub last_modified_by: String,
+    pub last_modified_at: NaiveDateTime,
```

Comment:
don't use `NaiseDateTime`, use `DateTime<Utc>` everywhere
we had to migrate all the time formats from `NaiveDateTime` to `DateTime<Utc>` due to smithy
```suggestion
    pub last_modified_at: DateTime<Utc>,
```